zstacklib/zstacklib/utils/upload_task.py

```
@@ -0,0 +1,409 @@
+
+        def get_long_field(key, default=None):
+            v = req_header.get(key, default)
+            try:
+                lv = long(v)
+                if lv < 0:
+                    raise ValueError
+                return lv
+            except ValueError:
+                raise Exception('invalid header "%s": %s' % (key, v))
+
+        up = upload_task.UploadParam()
```

Ruff (0.13.1)

Undefined name `upload_task`

(F821)